src/Microsoft.Azure.Functions.ExtensionBundle/bundleConfig.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
     "bundleId":  "Microsoft.Azure.Functions.ExtensionBundle.Preview",
-    "bundleVersion":  "4.16.0",
+    "bundleVersion":  "4.17.0",
     "templateVersion":  "4.0.2964",

```

Comment:
Yes, but the template version is already 2964 🤔

Reply:
@fabiocav do you know what updates the template version? I see other bundle updates PR's where both the bundle version and the template version are updated. Not sure why is it not updating in this PR

Reply:
Are you expecting to reference a new template version?

Reply:
I'm trying to get my new change https://github.com/Azure/azure-functions-extension-bundles/pull/364 in a bundle. Since this is a new update to the extensions, I was expecting a new template version as compared to other bundle updates.

Reply:
Where are your updated templates? Have you updated? Them?

Reply:
There was no template updates. But also the other bundle update PR's had no template updates, but I still see the template version getting updated.

Reply:
Spoke to Fabio offline. We don't need the template version update since there are no new templates.

Reply:
Had a sync with @gavin-aguiar , if there's no expectation we'll need a new set of templates, we're fine with the current state.